recipe/meta.yaml

```diff
@@ -0,0 +1,57 @@
+{% set name = "laszip-python" %}
+{% set version = "0.2.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/tmontaigu/laszip-python/archive/refs/tags/{{ version }}.tar.gz
+  sha256: 31572639a928fabf7d60a340970c6ae256ae9382e0420e938d6ab237553ac57b
+  patches:
+    - patches/0001-Add-missing-includes.patch
+
+build:
+  number: 0
+  script:
+    - export CMAKE_GENERATOR="Ninja"  # [not win]
```

Comment:
```suggestion
    - export CMAKE_BUILD_TYPE="Release"  # [not win]
    - set CMAKE_BUILD_TYPE=Release # [win]
    - export CMAKE_GENERATOR="Ninja"  # [not win]
```

Reply:
`scikit-build-core`'s default is already Release. See https://scikit-build-core.readthedocs.io/en/latest/configuration.html#full-schema (cmake/build-type). Additionally, `scikit-build-core` consumes the `CMAKE_ARGS` environment variable (which is set by our compiler activation feedstocks and already contains `Release` (at least on Linux and macOS.

Reply:
You are right, for `[not win]` goes to `Release`
For `win` I can see:
```
...
  -- Build files have been written to: C:/Users/lorepirri/AppData/Local/Temp/tmpjw3zaa90/build
  *** Building project with Ninja...
  [1/4] Building CXX object CMakeFiles\laszip.dir\src\laszipper.cpp.obj
  [2/4] Building CXX object CMakeFiles\laszip.dir\src\lasunzipper.cpp.obj
  [3/4] Building CXX object CMakeFiles\laszip.dir\src\laszip.cpp.obj
  [4/4] Linking CXX shared module laszip.cp38-win_amd64.pyd
  *** Installing project into wheel...
  -- Install configuration: "Debug"
  -- Installing: C:\Users\LOREPI~1\AppData\Local\Temp\tmpjw3zaa90\wheel\platlib/laszip/laszip.cp38-win_amd64.pyd
  -- Installing: C:\Users\LOREPI~1\AppData\Local\Temp\tmpjw3zaa90\wheel\platlib/laszip/laszip3.dll
  *** Making wheel...
  *** Created laszip-0.2.3-cp38-cp38-win_amd64.whl...
  Building wheel for laszip (pyproject.toml): finished with status 'done'
...
```
But since we don't build for `win` we can ignore this

Reply:
Weird.

Reply:
better to be explicit ?

Reply:
I pushed a commit that forces the mode to be `Release`.